docs/source/07-02-troubleshooting-tips.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Troubleshooting Tips
+permalink: /troubleshooting-tips/
+---
+
+## I've installed Talk but I can't see the comment stream appear on my articles
+
+* Make sure you've adding the correct domains to your Permitted Domains in Configure > Tech Settings
+* Make sure you've correctly added the embed via your CMS to your article pages
+* Check the console for any errors and you can file a bug via [support](mailto:support@coralproject.net) if you can't resolve the issue 
+
+## My commenters are reporting they can't see the comment stream, but I'm able to
+
+* If this seems to be isolated to one commenter, it could be related to 3rd party cookies 
+* Post 4.x Talk doesn't require 3rd party cookies to be enabled. Check the version of Talk you're using and you might consider upgrading. 
+* A quick fix in the meantime is to ask them to allow 3rd party cookies
+* You could also try asking them to clear their browser cache
+
+
+## My commenters are reporting that they cannot login to Talk
+
+If you're using your own custom auth plugin: 
+
+* Review the code and your server logs to ensure your plugin is working correctly. Check [our auth docs](https://docs.coralproject.net/talk/integrating/authentication/) for more tips.
```

Comment:
Replace with relative link "/talk/..." instead of "https://..."

Reply:
Fixed!